timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -238,6 +349,39 @@ public void throwsOnFastForwardWithIncorrectParameter() throws IOException {
         assertThat(response.code()).isEqualTo(HttpStatus.BAD_REQUEST_400);
     }
 
+    private List<Future<LockRefreshToken>> requestLocks(RemoteLockService lockService, int number, ExecutorService executorService) {
+        List<Future<LockRefreshToken>> futures = Lists.newArrayList();
+        for (int i = 0; i < number; i++) {
+            int currentTrial = i;
+            futures.add(executorService.submit(() -> lockService.lock(String.valueOf(currentTrial), SLOW_REQUEST)));
+        }
+        return futures;
+    }
+
+    private int unlockAndCountExceptions(RemoteLockService lockService, List<Future<LockRefreshToken>> futures) {
+        final int[] exceptionCounter = {0};
```

Comment:
I get that you're doing this so that `exceptionCounter` is final which you need for the forEach(), but I think I'd prefer to use `AtomicInteger` here